docs/console/project-configuration/registry-configuration.md

```diff
@@ -0,0 +1,32 @@
+---
+id: registry-configuration
+title: Docker Registry Configuration
+sidebar_label: Project Configuration
+---
+
+The Docker registry is a server side application that stores and lets you distribute your docker images.
+In order to pull and push images and use them in different pipelines it is necessary to configure it properly.
+To gain access to private registries it is necessary to create and configure your secrets.
+
+### Step 1
+Go in the setting interface of the console, on the Workload & Runtime section it is possible to add and modify a secret.
```

Comment:
Let's try to be more specific.

Reply:
Is a specific permission required to do this? I assume so. Maybe we should add a warning to mention it!